calicoctl/commands/node/permissions.go

```diff
@@ -12,22 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package ipam
+package node
 
 import (
 	"fmt"
-	"net"
 	"os"
 )
 
-// validateIP takes a string as an input and makes sure it's a valid IPv4 or IPv6 address.
-func validateIP(str string) net.IP {
-	// Parse the input string as an IP address (IPv4 or IPv6).
-	// This also validates the IP address.
-	ip := net.ParseIP(str)
-	if ip == nil {
-		fmt.Println("Invalid IP address specified.")
+func enforceRoot() {
```

Comment:
Well yes, except that utils packages are frowned upon. 

I have renamed the docoptutils to be argils -- so it is clearer that it is about argument utilities.
The enforceRoot() - I'm happy with this being in the node section since we will only want to enforce root on the node commands - all other commands should just be wrappers around etcd.